src/components/NcModal/NcModal.vue

```diff
@@ -916,14 +916,14 @@ export default {
 				box-sizing: border-box;
 				width: var(--default-clickable-area);
 				height: var(--default-clickable-area);
-				margin: calc(calc($header-height - var(--default-clickable-area)) / 2);
+				margin: calc(calc(var(--header-height) - var(--default-clickable-area)) / 2);
 				cursor: pointer;
 				opacity: $opacity_normal;
 			}
 		}
 
 		&:deep() .action-item {
-			margin: calc(calc($header-height - var(--default-clickable-area)) / 2);
+			margin: calc(calc(var(--header-height) - var(--default-clickable-area)) / 2);
```

Comment:
Just a late thought, we don't need double `calc` here, right? Should work with a single one

Reply:
Yes 

```suggestion
			margin: calc((var(--header-height) - var(--default-clickable-area)) / 2);
```